pkg/asset/agent/mirror/mirror.go

```diff
@@ -55,3 +58,31 @@ func (m *AgentMirror) Load(f asset.FileFetcher) (bool, error) {
 
 	return false, nil
 }
+
+// GetReleaseImageMirror returns the mirror corresponding to the release image
+func (m *AgentMirror) GetReleaseImageMirror(releaseImage string) string {
+	// If registries.conf is defined, find mirror stripping the version out, e.g. for
+	// registry.ci.openshift.org/ocp/release:4.11.0-0.nightly-2022-06-06-201913, use
+	// registry.ci.openshift.org/ocp/release
+	source := strings.Split(":", releaseImage)
+
+	for _, file := range m.FileList {
```

Comment:
I'm not sure about the reason of having this portion code as a separate asset. Wouldn't be easier to move it into the  `RegistriesConf` asset and expose the mirror string as an asset member?

Reply:
Changed to add function in registriesconfig asset that will build the source/mirror mapping and can be used with the registry  to find the mirror. Function was copied from assisted-service to avoid importing entire pkg.